src/tests/ftest/performance/mdtest_easy.py

```diff
@@ -34,6 +35,16 @@ def test_performance_mdtest_easy_dfs_ec_16p2g1(self):
         """
         self.run_performance_mdtest(namespace="/run/mdtest_dfs_ec_16p2g1/*")
 
+    def test_performance_mdtest_easy_dfuse_s1(self):
+        """Test Description: Run MDTest Easy, POSIX dfuse, S1.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,large
+        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse
+        :avocado: tags=performance_mdtest_easy_dfuse_s1,dfuse
```

Comment:
The tags are usually in order (per line) of frequency, stage, feature, unique tags. Could we please move dfuse to line 3? Either of these
```
        :avocado: tags=all,manual
        :avocado: tags=hw,large
        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse,dfuse
        :avocado: tags=performance_mdtest_easy_dfuse_s1
```
```
        :avocado: tags=all,manual
        :avocado: tags=hw,large
        :avocado: tags=dfuse
        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse
        :avocado: tags=performance_mdtest_easy_dfuse_s1
```